packages/syntax-floating-components/src/Tooltip/Tooltip.module.css

```diff
@@ -0,0 +1,17 @@
+.tooltipContainer {
+  /* TODO make into default var --tooltip */
+  z-index: 1100;
```

Comment:
@JFChoe By default we should not set a zIndex, that should be a prop we allow to be passed in.

Reply:
@JFChoe seems like we still set the zIndex, can we just pass that in as a prop?

Reply:
@JFChoe seems like we do pass it in as a prop but also set it here, let's remove it in the CSS